src/lib/parameters/px4params/markdownout.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, groups):
                         min_val='?'
                     if not max_val:
                         max_val='?'
-                    max_min_combined+='%s > %s ' % (min_val, max_val)
+                    max_min_combined+='%s, %s ' % (min_val, max_val)
```

Comment:
```suggestion
                    max_min_combined+='[%s, %s] ' % (min_val, max_val)
```

Reply:
I prefer this. Let's try it out. Can always revert.